modules/grpc/otel/otel-protobuf-parser.cpp

```diff
@@ -313,6 +323,8 @@ _parse_metadata(LogMessage *msg)
 
   /* .otel.scope.attributes */
   _add_repeated_KeyValue_fields(msg, ".otel.scope.attributes", scope.attributes());
+  if (set_hostname)
+    _set_hostname_from_attributes(msg, scope.attributes());
```

Comment:
my understanding is that host.name is a resource attribute, isn't it?

why extract it from both the log_record and the scope?

Reply:
Yes, the spec is about the Resource level. I wanted to make sure and extract it from every possible location, but I should probably remove them as this is not practical.

Also:
https://opentelemetry.io/docs/specs/otel/logs/data-model-appendix/